third-party/realdds/src/dds-device-impl.cpp

```diff
@@ -258,6 +254,58 @@ void dds_device::impl::on_known_notification( nlohmann::json const & j )
 }
 
 
+void dds_device::impl::on_log( nlohmann::json const & j )
+{
+    // This is the notification for "log"  (see docs/notifications.md#Logging)
+    //     - `entries` is an array containing 1 or more log entries
+    auto it = j.find( entries_key );
+    if( it == j.end() )
+        throw std::runtime_error( "log entries not found" );
```

Comment:
I don't think the client need to throw if the server (or someone) sent an illegal log. Should print a warning to log.

Reply:
It gets caught and logged, nothing else